core/src/main/java/org/opensearch/sql/analysis/AnalysisContext.java

```diff
@@ -13,26 +12,22 @@
 import org.opensearch.sql.expression.NamedExpression;
 import org.opensearch.sql.expression.function.FunctionProperties;
 
-/**
- * The context used for Analyzer.
- */
+/** The context used for Analyzer. */
 public class AnalysisContext {
-  /**
-   * Environment stack for symbol scope management.
-   */
+  /** Environment stack for symbol scope management. */
   private TypeEnvironment environment;
-  @Getter
-  private final List<NamedExpression> namedParseExpressions;
 
-  @Getter
-  private final FunctionProperties functionProperties;
+  @Getter private final List<NamedExpression> namedParseExpressions;
```

Comment:
https://github.com/diffplug/spotless/blob/main/plugin-gradle/README.md#formatannotations Spotless says  that `Type annotations should be on the same line as the type that they qualify.`. I think it might be mandatory, but I am investigating.

Reply:
https://github.com/google/google-java-format/issues/648#issuecomment-1404405114 Looks like it is still in Java Google Formatter. :(

Reply:
This doesn't change class annotations though.
Somewhere we have multiple annotations for class members. What happens here?
https://github.com/Bit-Quill/opensearch-project-sql/blob/6c3744ec81931567d50f47075cd45b3e8906997c/plugin/src/main/java/org/opensearch/sql/plugin/transport/TransportPPLQueryRequest.java#L38-L41

Reply:
Looks like on that it does not move anything. I think to get fully correct formatting we would need to enable this which can be done in a later PR. https://github.com/diffplug/spotless/blob/main/plugin-gradle/README.md#formatannotations